pandas/tests/series/test_io.py

```diff
@@ -162,6 +162,16 @@ def test_to_csv_compression(self, compression):
                                                    index_col=0,
                                                    squeeze=True))
 
+    def test_from_csv_dtype_str(self):
```

Comment:
* This test should really belong in the `tests/io/parser/na_values` tests
* Do we really need to round-trip to surface this bug?  Can we try to just use `StringIO`?